Gems/NvCloth/Assets/Objects/cloth/Environment/Cloth_Blinds.fbx.assetinfo

```diff
@@ -28,7 +24,7 @@
                     },
                     {
                         "$type": "ClothRule",
-                        "meshNodeName": "RootNode.pPlane1.pPlane1_1",
+                        "meshNodeName": "RootNode.pPlane1",
```

Comment:
I don't know what's causing that many changes in the cloth assets. But I've noticed that the nodes names have changed, which can affect the node selected in Cloth Components in levels. So my initial worry is that the levels of automated cloth tests (located here `o3de\AutomatedTesting\Levels\NvCloth`) might not be setup correctly anymore.

I'll sync to the branch to verify the assets. I'll try to do it today, but it most likely happen tomorrow.